lib/cretonne/src/ir/instructions.rs

```diff
@@ -235,6 +236,12 @@ pub enum InstructionData {
         src: RegUnit,
         dst: RegUnit,
     },
+    CopySpecial {
```

Comment:
Whenever I add a new instruction format, I also add a test case to `filetests/parser/tiny.cton` which then covers both the printing and parsing of the new format.